### ISSUE TITLE
Adjust building names translations

### ISSUE THREAD

**Issue by NaturallyCreepy** [DexterHuang/CyberCodeOnline#3124]
Hello, after you enabled the italian translation for procedural names, I realized the order of the words in building names is hard-coded: adjectives are always placed before the name. In italian (french, spanish also) adjectives are usually put after the name of the subject.

Could you add a way to adjust prefixes like you did for equipment names? I mean something like this:

<pre lang="json"><code>"trans": "<b>${name}</b> ${part1} ${part2}"
"eng": "${part1} ${part2} <b>${name}</b>"</code></pre>

Thanks 😄

**Comment by NaturallyCreepy** [comment]
For example, the following building in Suzawa station is not correct:
![immagine](https://user-images.githubusercontent.com/34171954/216770077-f064f200-a68b-47e1-bf68-fd030cd4500c.png)

It should be: "Pasticceria Cromatico/a di Suzawa"

In order to have that result, I would need to be able to have a translation scheme such as:
<pre><code>"trans": "<b>${buildingName} ${prefix} di ${stationName}</b>"
"eng": "${prefix} ${stationName} ${buidingName}"</code></pre>

**Comment by hkcisonline** [comment]
I agree with NaturallyCreepy about the Italian et French translation (for my part), if we could have a French translation scheme such as 

"trans": "${buildingName} ${prefix} de ${stationName}"
"eng": "${prefix} ${stationName} ${buildingName}"

it would be great as it took really a bunch of work to translate and propose all those localized names ;)

TIA
